### ISSUE TITLE
Extern led is not switched on

### ISSUE THREAD

**Issue by martienverbeek** [ukagit/iambic-keyer#1]
The extern_led is defined on port 3 and also switched to 0, but where the intern_led is switched to 1 the extern_led is not swithed to 1.

**Comment by ukagit** [comment]
Hi ich melde mich in 1-2 Tagen muss mir den Code nochmal anschauen …73 Ulrich Am 10.03.2023 um 19:22 schrieb martienverbeek ***@***.***>:﻿
The extern_led is defined on port 3 and also switched to 0, but where the intern_led is switched to 1 the extern_led is not swithed to 1.

**Comment by ukagit** [comment]
Hi MartinI did an UpdatePlease Check73 UlrichAm 10.03.2023 um 19:22 schrieb martienverbeek ***@***.***>:﻿
The extern_led is defined on port 3 and also switched to 0, but where the intern_led is switched to 1 the extern_led is not swithed to 1.

**Comment by martienverbeek** [comment]
Yes, that is what I did as well. Thanks in name of the community!👍

**Comment by ukagit** [comment]
hi marin,
thanks for feedback.

i personal do not use the kind of keyer , it was only for fun.

I extended the micropython code for „sensor" keyer and small display for WPM change, the processor is then the esp32.
regards 73 ulrich

**Comment by martienverbeek** [comment]
I use this IAMBIC keyer as simple training device for morse for several months. To recapture the speed I joined a dutch VERON training group. Last Tuesday our very experienced morse guru (50 wpm after 40 years of CW experience) was enthusiastic about this pico keyer.

**Comment by ukagit** [comment]
Hello Martien,

I'm excited to share that I've incorporated two new features into the new ESP32 keyer: the "ration of dit-time" and the "length of dat time." I am currently working on integrating the correct words per minute (wpm) adjustment, which I plan to commit to github after the holiday season.

I believe these keyer settings might be of interest to your CW expert. 

Take a moment to check out the README on GitHub for more details: https://github.com/ukagit/iambic-keyer-esp32-touchpad <https://github.com/ukagit/iambic-keyer-esp32-touchpad>.

If you find these features relevant, I'm also open to porting the code from ESP32 to Pico. Let me know your thoughts.

Wishing you a wonderful christmas time!

Best regards,

Ulrich




> Am 17.12.2023 um 22:24 schrieb martienverbeek ***@***.***>:
> 
> I use this IAMBIC keyer as simple training device for morse for several months. To recapture the speed I joined a dutch VERON training group. Last Tuesday our very experienced morse guru (50 wpm after 40 years of CW experience) was enthusiastic about this pico keyer.
> 
>
